prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/SchemaUtils.kt

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.utils
+
+/**
+ * helper
+ */
+fun getURI(folder: String?, schemaName: String): String {
```

Comment:
I don't think this should be required... the class path should contain all the info about the found and we shouldn't be constructing it dynamically.  I think you're editing the old code path that supports relative pathing.

Reply:
yup, adapting old code with URI semantics....

Reply:
See comment above, but the old code _should_ be able to exist along side this new URI pattern, but let me know if that isn't quite working out